internal/rest/avi_obj_vs.go

```diff
@@ -230,7 +234,16 @@ func (rest *RestOperations) AviVsSniBuild(vs_meta *nodes.AviVsNode, rest_method
 		pool_ref := "/api/pool/?name=" + vs_meta.DefaultPool
 		sniChild.PoolRef = &pool_ref
 	}
-	var rest_ops []*utils.RestOp
+
+	var datascriptCollection []*avimodels.VSDataScripts
+	for i, script := range vs_meta.VsDatascriptRefs {
+		j := int32(i)
```

Comment:
isn't the loop already initializing the value of 'I' and 'script' every time you iterate through it? Then why again re-init them into j and datascript?

Reply:
the `range` functionality does not re-initialize these variables. So if we take the address of this, it'll always end up  being the same.  https://stackoverflow.com/questions/48826460/using-pointers-in-a-for-loop